### ISSUE TITLE
Include public address in public endpoint certificates

### ISSUE THREAD

**Issue by gnarea** [relaynet/specs#83]
# Executive summary

This would save the app developer having to specify the address separately, and make endpoint libraries (like the one for Android) simpler.

# The solution you'd like

We could add this address to the Distinguished Name of the Subject (with a new, Relaynet-specific OID) or it could be added as a X.509 extension.

# Privacy/security considerations

Private or public gateways won't be able to identify the service by looking at the parcel, because public endpoint certificates are never visible to gateways. When public gateways send parcels, they use their PDA, which won't include the public address.

# Alternatives considered

Basically the workarounds considered in https://github.com/relaycorp/relaynet-endpoint-android/issues/47

**Comment by gnarea** [comment]
This is no longer an issue with the work done on https://github.com/relaycorp/relayverse/issues/19, since parcel recipients now include the Internet address (when the parcel is bound for the Internet).
